thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/v2/DataResource.java

```diff
@@ -179,6 +179,12 @@ public Long getMetricMaxDataTime(@PathParam("metricId") Long metricId) {
       Set<String> aliasParts = new HashSet<>(Arrays.asList(name.split("\\s+")));
       metricConfigs = metricConfigDAO.findWhereAliasLikeAndActive(aliasParts);
     }
+    Collections.sort(metricConfigs, new Comparator<MetricConfigDTO>() {
+      @Override
+      public int compare(MetricConfigDTO o1, MetricConfigDTO o2) {
+        return o1.getAlias().compareTo(o2.getAlias());
```

Comment:
Is sort by alias is what we expect, or sort by metric name?

Reply:
yes. alias is what the user sees.